coe_code/readme.md

```diff
@@ -138,4 +138,16 @@ The matched output table has the following columns:
    - `match_datetime` When the match was made
    - `run_id` An autogenerated run ID.
 
+## Adding additional information to matched output
+To add further detail to the matched output table (including additional lower level codes and descriptions of the codes) run the `format_output.py` script located in the coe_code folder.
```

Comment:
think they're higher level codes (although probably depends on which way you look at it)

Reply:
feel free to not change